global.json

```diff
@@ -29,10 +29,10 @@
       "allowPrerelease": false
     },
     "vswhere": "2.2.7",
-    "dotnet": "8.0.100-preview.6.23330.14"
+    "dotnet": "8.0.100-rtm.23506.1"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.23364.2",
+    "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.23528.2",
     "Microsoft.DotNet.Helix.Sdk": "8.0.0-beta.23226.4"
```

Comment:
I don't see a dependency bump for the Helix.Sdk. If you don't use Helix testing in this repository, I would remove this line.